### DISCUSSION TITLE
Assigning custom weapons to a class

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1571] [Question and answer]
Hello is it possible to assign weapons to a certain classes? Some of my snipers are listed as "Assault Rifles", or smgs as "Assault Rifles".

**Comment by Vdauphin** [comment]
Hello,

See https://github.com/Vdauphin/HeartsAndMinds/discussions/1548#discussioncomment-7933417

Which weapons?

Cheers

**Reply by XDSenDX** [reply]
Weapons from this mod https://steamcommunity.com/sharedfiles/filedetails/?id=3090136183

XM2010 from RHS.

**Reply by Vdauphin** [reply]
RHS should be able to fix it
Weapon can be split is a few kinds of types: https://community.bistudio.com/wiki/BIS_fnc_itemType

**Reply by XDSenDX** [reply]
But is there any ways i can fix it my own?

**Reply by Vdauphin** [reply]
Create a mod that fix the mod I think

**Reply by XDSenDX** [reply]
That will prob violate TOS then, i'll just leave it there for now.

**Reply by Vdauphin** [reply]
> That will prob violate TOS then

No lol

**Reply by XDSenDX** [reply]
> > That will prob violate TOS then
> 
> No lol

I'll just leave them like that for the moment. I'll consider this again in the future.

Thanks.
